CONTRIBUTING.md

```diff
@@ -1,4 +1,161 @@
 # Contribution Guide
 
-See the [Contribution Guide](https://github.com/pingcap/community/blob/master/CONTRIBUTING.md) in the
-[community](https://github.com/pingcap/community) repo.
+TiDB is a community-driven open source project and we welcome any contributor.  Contributions to the TiDB project are expected to adhere to our [Code of Conduct](https://github.com/pingcap/community/blob/master/CODE_OF_CONDUCT.md).
+
+This document outlines some conventions about development workflow, commit message formatting, contact points and other resources to make it easier to get your contribution accepted. This document is the canonical source of truth for things like supported toolchain versions for building and testing TiDB.
+
+<!-- TOC -->
+
+- [Contribution Guide](#contribution-guide)
+    - [Before you get started](#before-you-get-started)
+        - [Sign the CLA](#sign-the-cla)
+        - [Setting up your development environment](#setting-up-your-development-environment)
+    - [Your First Contribution](#your-first-contribution)
+        - [Hacktoberfest](#hacktoberfest)
+    - [Before you open your PR](#before-you-open-your-pr)
+    - [TiDB Contribution Workflow](#tidb-contribution-workflow)
+    - [Get a code review](#get-a-code-review)
+        - [Style reference](#style-reference)
+    - [Bot Commands](#bot-commands)
+    - [Benchmark](#benchmark)
+
+<!-- /TOC -->
+
+## Before you get started
+
+### Sign the CLA
+
+Click the **Sign in with Github to agree** button to sign the CLA. See an example [here](https://cla-assistant.io/pingcap/tidb?pullRequest=16303).
+
+What is [CLA](https://www.clahub.com/pages/why_cla)?
```

Comment:
The link is broken, use this one:https://en.wikipedia.org/wiki/Contributor_License_Agreement